attributes/default.rb

```diff
@@ -29,7 +30,7 @@
 
 default['consul']['service']['config_dir'] = join_path config_prefix_path, 'conf.d'
 
-default['consul']['version'] = '0.9.3'
+default['consul']['version'] = '1.4.4'
```

Comment:
Our default version is already 1.4.4 (in the old environment) so we're just bumping it here to match it up.